src/core/reducers/categories.js

```diff
@@ -96,6 +104,47 @@ export function createEmptyCategoryList(): EmptyCategoryListType {
   }, {});
 }
 
+export const getCategories = (
+  categoriesState: CategoryMapType,
+  appName: string,
+  addonType: string,
+): Array<mixed> | null => {
+  if (
```

Comment:
I guess we need `invariants` for all of the required arguments like we have in `getCategoryNames` below.

Reply:
Instead of `if` we have to use `invariants` like in this way:
```
  invariant(categoriesState, 'categories state can not be empty');
  invariant(categoriesState[appName], 'categories state does not contains the app name');
  invariant(categoriesState[appName][addonType], 'categories state does not contain the addon type');

  return Object.values(categoriesState[appName][addonType]);
```
Am I correct ?

Reply:
No. We should just have invariants for `categoriesState`, `appName` and `addonType`, because those are the required arguments. You can then keep the logic you have in the `if` statement for `categoriesState[appName] && categoriesState[appName][addonType]` because there is still a possibility that those will be empty/null.

Reply:
Oh correct! I will change this accordingly.